### ISSUE TITLE
error in EnumProcessModules

### ISSUE THREAD

**Issue by timmcd** [optix2000/totsugeki#23]
I have been using an older version of Totsugeki successfully. Today I downloaded the v1.1.2 release and immediately tried to run it, and got this error. I thought it sounded a bit like a permissions issue so I tried running it as administrator, to no avail.

```
 _____       _                             _     _
|_   _|___  | |_  ___  _   _   __ _   ___ | | __(_)
  | | / _ \ | __|/ __|| | | | / _` | / _ \| |/ /| |
  | || (_) || |_ \__ \| |_| || (_| ||  __/|   < | |
  |_| \___/  \__||___/ \__,_| \__, | \___||_|\_\|_|
                              |___/
                                         v1.1.2
Starting GGST...
Started Proxy Server on port 21611.
Waiting for GGST process...
Error at offset 0x0: error in EnumProcessModules: Only part of a ReadProcessMemory or WriteProcessMemory request was completed.
```

It also opens a separate window...

![image](https://user-images.githubusercontent.com/112259/128617346-83bc81d9-8be3-4ffe-9059-657ceec37362.png)

**Comment by timmcd** [comment]
v1.1.1 seems to work exactly as expected!

**Comment by optix2000** [comment]
I found this myself yesterday. It's an issue that happens when Totsugeki tries to patch GGST when the GGST process has started, but hasn't finished loading. Depending on the version of Totsugeki you use, the stars may or may not align to cause this issue (we're talking about a window of milliseconds).

This is fixed in 66e1d28aa7f6f2b7c87be73b395f33b337355777. I'll cut a new release soon that should fix this issue.

**Comment by optix2000** [comment]
After looking at this more, there's actually a bug in v1.1.2 caused by the graceful shutdown, where loop for watching GGST was running without delays which is why this triggers 1000x more often on v1.1.2.

Definitely stick with v1.1.1 for now.

Thanks for testing the beta release!
